src/renderer/remote.ts

```diff
@@ -311,7 +311,7 @@ function metaToError (meta: { type: 'error', value: any, members: ObjectMember[]
 
 function handleMessage (channel: string, handler: Function) {
   ipcRenderer.on(channel, (event, passedContextId, id, ...args) => {
-    if (event.senderId !== 0) {
+    if (event.senderId !== 0 && event.senderId !== undefined) {
```

Comment:
Or just `if (event.senderId > 0)`?

Reply:
This is less explicit, but if you prefer, I amend the patch accordingly.

Reply:
nah, let's leave it then :)